js/server.js

```diff
@@ -13,6 +14,10 @@ module.exports = config => {
 
   app.use(express.static(config.build))
 
+  app.post('/node_modules/brightspot-express/styleguide/*', function (req, res) {
```

Comment:
I don't think using this route/pattern will work in a project that's not brightspot-express